homeassistant/components/script/__init__.py

```diff
@@ -301,29 +301,28 @@ class ScriptEntity(ToggleEntity, RestoreEntity):
 
     icon = None
 
-    def __init__(self, hass, object_id, cfg, raw_config, blueprint_inputs):
+    def __init__(self, hass, key, cfg, raw_config, blueprint_inputs):
         """Initialize the script."""
-        self.object_id = object_id
         self.icon = cfg.get(CONF_ICON)
         self.description = cfg[CONF_DESCRIPTION]
         self.fields = cfg[CONF_FIELDS]
 
         # The object ID of scripts need / are unique already
         # they cannot be changed from the UI after creating
```

Comment:
```suggestion
        # The key of scripts need / are unique already
        # they cannot be changed from the UI after creating
```
One missing object id in the comment